source/styleEditor.js

```diff
@@ -1,3 +1,27 @@
 $(document).ready(function(){
   // your code goes here.
+  const form = document.querySelector('#style_editor');
+
+  const styleIt = evt => {
+    const selector = document.querySelector('input[name="selector"]').value;
+    const property = document.querySelector('input[name="property"').value;
+    const value = document.querySelector('input[name="value"]').value;
+
+    modifyEle(selector, property, value);
```

Comment:
You could also return these three values in an object or array, change `modifyEle` to accept that data type (object or array), and call `modifyEle` in your event listener.

The main goal of this approach is to de-couple your two functions, which makes them more flexible and easy to test.